File: SpaceInvaders/GameObject/Alien/SquidAlien.cs
using System;
using System.Diagnostics;

namespace SpaceInvaders
{
    class SquidAlien : Alien
    {
        int squidPoints = 30;

        public SquidAlien(GameObject.Name name, Sprite.Name spriteName, float posX, float posY)
            : base(name, spriteName)
        {
            this.x = posX;
            this.y = posY;
        }

        ~SquidAlien()
        {

        }

        public override int GetPoints()
        {
            return squidPoints;
        }

        public override void Update()
        {
            base.Update();
        }
    }
}

File: quarkus-migration/space-invaders-quarkus/src/main/java/net/beeland/spaceinvaders/gameobject/SquidAlien.java
package net.beeland.spaceinvaders.gameobject;

/**
 * SquidAlien - Top row alien, worth 30 points
 */
public class SquidAlien extends Alien {
    
    private static final int SQUID_POINTS = 30;
    
    /**
     * Constructor
     *
     * @param posX Initial X position
     * @param posY Initial Y position
     */
    public SquidAlien(float posX, float posY) {
        super(GameObject.GameObjectName.SQUID_ALIEN);
        this.x = posX;
        this.y = posY;
    }
    
    @Override
    public int getPoints() {
        return SQUID_POINTS;
    }
    
    @Override
    public Type getType() {
        return Type.SQUID;
    }
    
    @Override
    public void update() {
        super.update();
    }
}